test/studies/prk/DGEMM/SUMMA/dgemm.ml-execopts

```diff
@@ -0,0 +1 @@
+--order 10000 --iterations 20 --windowSize 10000 
```

Comment:
I want to use power-of-two sizes to avoid any noise due to load imbalance in our nightly performance testing. 

I have also observed a big performance drop going from 4096x4096 to 8192x8192 probably because we are allocating more than the cache per locale.

For nightly performance tracking purposes I think we should go with 4096x4096 and 10 iterations. It finishes under 30 seconds and doesn't have a lot of noise.

Note that the main purpose of these nightly tests is not answering "what is the DGEMM performance of Chapel?" but more "did we mess up DGEMM performance yesterday?" So, while 4096 might be a bit small, it is not too small and I think is suitable for capturing any unforeseen performance regressions.